drivers/usb/device/Kconfig

```diff
@@ -21,6 +21,21 @@ config USB_DW_IRQ_PRI
 	help
 	USB interrupt priority.
 
+config USB_DW_DMA
+	bool "Enable Designware USB Controller Buffer DMA mode"
+	default y
+	help
```

Comment:
This patch breaks my USB application (yet to be submitted), IMO this should not be enabled by default